apigentools/commands/init.py

```diff
@@ -16,6 +16,54 @@
 
 
 class InitCommand(Command):
+    def __init__(self, config, args):
+        self.config_file_json = {
+                                "codegen_exec": "openapi-generator",
+                                "languages": {},
+                                "server_base_urls": {
+                                    "v1": "https://api.myserver.com/v1",
+                                },
+                                "spec_sections": {
+                                    "v1": [],
+                                },
+                                "spec_versions": ["v1"],
+                            }
+        self.v1_header_json = {
+                                "info": {
+                                    "contact": {},
+                                    "description": "Collection of all public API endpoints.",
+                                    "title": "My API endpoints",
+                                    "version": "1.0"
+                                },
+                                "openapi": "3.0.0",
+                            }
+        self.v1_shared_json = {
+                                "components": {
+                                    "schemas": {},
+                                    "parameters": {},
+                                    "securitySchemes": {},
+                                    "requestBodies": {},
+                                    "responses": {},
+                                    "headers": {},
+                                    "examples": {},
+                                    "links": {},
+                                    "callbacks": {},
+                                },
+                                "security": [],
+                                "tags": [],
+                            }
+        self.gitignore = [
```

Comment:
Similar to dicts, please indent the list entries by one indentation level more than the line with variable definition, e.g.

```python
mylist = [
    "foo",
    "bar",
]
```